theme/login/theme.properties

```diff
@@ -1,6 +1,6 @@
 parent=base
 import=common/keycloak
-styles=css/beagle.css css/styles.css css/material-design-iconic-font.min.css
+styles=css/beagle.css css/styles.css css/material-design-iconic-font.min.css css/font-awesome/css/font-awesome.min.css
```

Comment:
@mircowidmer: is the whole font-awesome stuff really needed? It feels like a bit to much ballast to add.

Reply:
@chloesoe I replaced the font awesome icon with the mdi, which is already included in the beagle theme.